snippets/logging-lokistack-cr-snip.adoc

```diff
@@ -0,0 +1,32 @@
+// Text snippet included in the following assemblies:
+// logging/logging-loki-ocp.adoc
+//
+// Text snippet included in the following modules:
+//
+//
+:_content-type: SNIPPET
+
+[source,yaml]
+----
+apiVersion: loki.grafana.com/v1
+kind: LokiStack
+metadata:
+  name: logging-loki
+  namespace: openshift-logging
+spec:
+  size: 1x.small <1>
+  storage:
+    schemas:
+    - version: v12
+      effectiveDate: "2022-06-01"
+    secret:
+      name: logging-loki-s3 <2>
+      type: s3 <3>
```

Comment:
I think we should mention `storage.tls` too, details are in https://github.com/openshift/loki/blob/release-5.7/operator/bundle/openshift/manifests/loki.grafana.com_lokistacks.yaml#L535-L552.

Reply:
This is a snippet for limited reuse, we have sections of using TLS.